third_party/pyth/multisig-wh-message-builder/tsconfig.json

```diff
@@ -1,15 +1,14 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "target": "es2016",
+    "target": "es6",
     "module": "commonjs",
-    "outDir": "lib",
+    "outDir": "dist",
```

Comment:
I think `lib` is the common output directory and also in package.json contents of `lib` are specified to be considered as the package.

Reply:
oh, I decided to do this because this is not a library package

Reply:
Aha I see, then it makes sense.